test/http_SUITE.erl

```diff
@@ -307,6 +307,20 @@ echo_body(Config) ->
 	end || Size <- lists:seq(MTU - 500, MTU)],
 	ok.
 
+%% Check if sending request whose header name is bigger than 64 bytes causes 431
+echo_body_max_header_name_length(Config) ->
+	ConnPid = gun_open(Config),
+	Ref = gun:post(ConnPid, "/echo/body", [{binary:copy(<<$a>>, 32768), <<"bad">>}], << "echo=name" >>),
```

Comment:
Most limits in Cowboy are soft limits. If we have already read the data from the socket and this data happened to be over the limit, we don't throw an error. This limit is about not reading endlessly from the socket.